### DISCUSSION TITLE
Add support for visualizing COG and STAC

### DISCUSSION THREAD

**Discussion by giswqs** [developmentseed/lonboard#201] [Ideas]
It would be awesome if lonboard can support visualizing raster datasets, such as COG and STAC. This can make lonboard a one-stop-shop for visualizing geospatial data. 

https://github.com/kylebarron/deck.gl-raster

**Comment by giswqs** [comment]
deck.gl supports xyz and wms tile layers. Would be great to add support for them as well. 

https://deck.gl/examples/tile-layer
https://deck.gl/examples/wms-layer

**Comment by kylebarron** [comment]
> It would be awesome if lonboard can support visualizing raster datasets, such as COG and STAC. This can make lonboard a one-stop-shop for visualizing geospatial data.
> 
> https://github.com/kylebarron/deck.gl-raster

This would be nice, but deck.gl-raster is very out of date right now, and it would be a significant chunk of work to update that and integrate it with lonboard. It would be ideal to find a funding arrangement to support that.

**Comment by sheecegardezi** [comment]
> It would be awesome if lonboard can support visualizing raster datasets, such as COG and STAC. This can make lonboard a one-stop-shop for visualizing geospatial data.
> 
> https://github.com/kylebarron/deck.gl-raster

This will be of intrest: https://github.com/gisat-panther/deck.gl-geotiff
